quarkus-reactive-kafka-consumer/src/main/resources/application.properties

```diff
@@ -6,17 +6,18 @@ quarkus.log.console.color=true
 quarkus.http.port=8083
 
 # all configuration to connect to kafka
-mp.messaging.connector.smallrye-kafka.security.protocol=SASL_SSL
-mp.messaging.connector.smallrye-kafka.ssl.protocol=TLSv1.2
-mp.messaging.connector.smallrye-kafka.sasl.mechanism=SCRAM-SHA-512
-mp.messaging.connector.smallrye-kafka.sasl.jaas.config=org.apache.kafka.common.security.scram.ScramLoginModule required username=${KAFKA_SCRAM_USERNAME} password=${KAFKA_SCRAM_PASSWORD};
-mp.messaging.connector.smallrye-kafka.ssl.truststore.location=${KAFKA_SSL_TRUSTSTORE_LOCATION}
-mp.messaging.connector.smallrye-kafka.ssl.truststore.password=${KAFKA_SSL_TRUSTSTORE_PASSWORD}
-mp.messaging.connector.smallrye-kafka.ssl.truststore.type=PKCS12
+%prod.kafka.bootstrap.servers=kafka:9092
+%prod.mp.messaging.connector.smallrye-kafka.security.protocol=SASL_SSL
+%prod.mp.messaging.connector.smallrye-kafka.ssl.protocol=TLSv1.2
+%prod.mp.messaging.connector.smallrye-kafka.sasl.mechanism=SCRAM-SHA-512
+%prod.mp.messaging.connector.smallrye-kafka.sasl.jaas.config=org.apache.kafka.common.security.scram.ScramLoginModule required username=${KAFKA_SCRAM_USERNAME} password=${KAFKA_SCRAM_PASSWORD};
+%prod.mp.messaging.connector.smallrye-kafka.ssl.truststore.location=${KAFKA_SSL_TRUSTSTORE_LOCATION}
+%prod.mp.messaging.connector.smallrye-kafka.ssl.truststore.password=${KAFKA_SSL_TRUSTSTORE_PASSWORD}
+%prod.mp.messaging.connector.smallrye-kafka.ssl.truststore.type=PKCS12
 # configuration to read messages
-mp.messaging.connector.smallrye-kafka.bootstrap.servers=${KAFKA_BOOTSTRAP_SERVERS}
-mp.messaging.incoming.my-data-stream.connector=smallrye-kafka
-mp.messaging.incoming.my-data-stream.topic=${KAFKA_TOPIC}
-mp.messaging.incoming.my-data-stream.value.deserializer=org.apache.kafka.common.serialization.DoubleDeserializer
+#mp.messaging.connector.smallrye-kafka.bootstrap.servers=${KAFKA_BOOTSTRAP_SERVERS}
```

Comment:
Removed this line since it gets defaulted to the proper value if it doesn't exist.